src/main/java/com/mojang/datafixers/types/DynamicOps.java

```diff
@@ -153,10 +153,8 @@ default T createLongList(final LongStream input) {
 
     default Optional<T> getGeneric(final T input, final T key) {
         return getMapValues(input).flatMap(map -> {
-            if (map.containsKey(key)) {
-                return Optional.of(map.get(key));
-            }
-            return Optional.empty();
+            T value = map.get(key);
+            return value != null ? Optional.of(value) : Optional.empty();
```

Comment:
Tiny nitpick: can be replaced with `Optional.ofNullable`.

Reply:
updated :)